model.py

```diff
@@ -7,6 +7,7 @@
 
 class Run_Metrics(SQLModel, table=True):
     __tablename__ = 'run_metrics'
+    __table_args__ = {'extend_existing': True}
```

Comment:
why did you add this?

Reply:
Every single time I refreshed it, it failed because Run_Metrics was already loaded. This fixes it
![Screenshot from 2022-07-08 11-59-48](https://user-images.githubusercontent.com/46976761/178028544-f642f32b-c9ca-455f-9112-9f6663d5830c.png)
.

Reply:
awesome! I tried several changes to fix that.